plugins/org.jkiss.dbeaver.ext.import_config/src/org/jkiss/dbeaver/ext/import_config/wizards/sqlworkbench/ConfigImportWizardPageSqlWorkbenchConnections.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.import_config.wizards.sqlworkbench;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.ext.import_config.wizards.*;
+import org.jkiss.dbeaver.model.connection.DBPDriver;
+import org.jkiss.dbeaver.model.impl.net.SSLHandlerTrustStoreImpl;
+import org.jkiss.dbeaver.model.net.DBWHandlerConfiguration;
+import org.jkiss.dbeaver.model.net.ssh.SSHConstants;
+import org.jkiss.dbeaver.registry.driver.DriverUtils;
+import org.jkiss.dbeaver.registry.network.NetworkHandlerDescriptor;
+import org.jkiss.dbeaver.registry.network.NetworkHandlerRegistry;
+import org.jkiss.utils.CommonUtils;
+import org.jkiss.utils.xml.XMLException;
+import org.jkiss.utils.xml.XMLUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.io.*;
+import java.util.Collection;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class ConfigImportWizardPageSqlWorkbenchConnections extends ConfigImportWizardPage {
+    private static final Log log = Log.getLog(ConfigImportWizardPageSqlWorkbenchConnections.class);
+    private final ImportDriverInfo mysqlDriver;
+    private final Set<String> keyWords;
+
+    //for matching like: db1.dev.dbeaver.com:22
+    private final Pattern urlPattern = Pattern.compile("^(?<host>[a-zA-Z0-9.-]+):(?<port>\\d+)$");
+
+    protected ConfigImportWizardPageSqlWorkbenchConnections() {
+        super("Connections");
+        setTitle("Connections");
+        setDescription("Import connections");
```

Comment:
Please add localization

Reply:
Currently, none of our import configurations are localized. It could be weird to have such only for one of them. I guess there should be a separate task for the localization of all import configuration

Reply:
That's a pity. Could you create this task then?

Reply:
[here](https://github.com/dbeaver/dbeaver/issues/34993)